src/backend/marsha/core/models/video.py

```diff
@@ -263,6 +262,16 @@ class Video(BaseFile, RetentionDateObjectMixin):
         null=True,
         blank=True,
     )
+    duration = models.FloatField(
+        help_text=_("Duration of the video in seconds"),
+        null=True,
+        blank=True,
+    )
+    size = models.FloatField(
+        help_text=_("Size of the video in bytes"),
+        null=True,
+        blank=True,
+    )
```

Comment:
FMI, why using floats here ? To ease future calculations ?

Reply:
To be honest I hesitate a lot with Integer. IMO use Integer is good enough